client/client.go

```diff
@@ -39,6 +39,7 @@ type VanClient struct {
 	DynamicClient   dynamic.Interface
 	DiscoveryClient *discovery.DiscoveryClient
 	LinkHandler     domain.LinkHandler
+	EventRecorder   kube.SkupperEventRecorder
```

Comment:
Is this the right place to hold the recorder? At present it is intended to be used from the service controller. Would having it as a member of the controller be better?

Reply:
There are two k8s events that are sent outside the service-controller, in the service-sync; when the connection is established with the sender or the receiver.

Reply:
service sync is logically part of the service controller. Perhaps we could pass in an event handler when creating the service sync instance?

Reply:
Updated in 16e762d1840cea04e30984d9be7b9ea58e89acdc